addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -154,6 +155,23 @@ function clickOnEdit(position = "bottom") {
     };
 }
 
+/**
+ * Click on the top right edit button and wait for the edit mode
+ * @param {*} position Where the purple arrow will show up
+ */
+function clickOnEditAndWaitEditMode(position = "bottom") {
+    return [{
+        trigger: "a[data-action=edit]",
+        content: _t("<b>Click Edit</b> to start designing your homepage."),
```

Comment:
Content always first please in test steps: the goal if to be able to human-read what a test does directly by reading the first line of every steps :+1: